dockerfiles/pytorch-dev/install_rocm_from_tarballs.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# -----------------------------------------------------------------------------
+# install_rocm_from_tarballs.sh
+#
+# Download and install ROCm tarballs for specified AMDGPU targets from
+# TheRock's nightly GitHub releases.
+#
+# Usage:
+#   INSTALL_PREFIX=/pathToTheRock VERSION=6.4.0rc20250420 ./install_rocm_from_tarballs.sh "gfx942 gfx1100"
```

Comment:
Tarballs are an implementation detail here. How about "install_rocm_from_release" for the script name? Then maybe "release version" as the option name instead of just "release" for more clarity? Python argparse could also support multiple names for the same option.

Reply:
I think `release_version` makes it clear - updated